src/app/dashboard/dashboard.tsx

```diff
@@ -14,7 +14,19 @@ export async function Dashboard({
   TESTING_FLAG?: boolean;
 }) {
   const user = (await dataModel.getUser(userID)) as User;
-  const userBudgets = await dataModel.getBudgetsForUser(userID);
+  const sgaUser = userIsSGA(user);
+  const budgets = await dataModel.getBudgets(
+    sgaUser ? [] : [new Filter('user_id', '==', userID)]
+  );
```

Comment:
the sga dashboard should still get budgets and not get an empty list; the sga dashboard should see all budget's with the 'submitted' status

Reply:
The empty list is the list of filters, so this would get all budgets for the sga user.  Although you're right that I forgot I would need to filter by status = submitted.